lib/apiservers/engine/backends/volume.go

```diff
@@ -67,50 +69,119 @@ func NewVolumeModel(volume *models.VolumeResponse, labels map[string]string) *ty
 type Volume struct {
 }
 
+// acceptedVolumeFilters are filters that are supported by VIC
+var acceptedVolumeFilters = map[string]bool{
+	"dangling": true,
+	"name":     true,
+	"driver":   true,
+}
+
```

Comment:
docker 1.13 adds support for label filtering -- you should either add that now or at a minimum have it as an unacceptedVolumeFilter